schemas/1.3.0/adaptive-card.json

```diff
@@ -830,6 +830,11 @@
               "type": "boolean",
               "description": "If `true`, displays the text with strikethrough."
             },
+            "underline": {
+              "type": "boolean",
+              "description": "If `true`, displays the text underlined.",
+              "default": false
+            },
```

Comment:
This file is generated from the files under schemas\src so you'll want to make the change there and regenerate this so it doesn't get lost next time we generate the schema.

Reply:
Actually, I'm being silly, i actually have a PR out for the schema changes already. [https://github.com/microsoft/AdaptiveCards/pull/4559](https://github.com/microsoft/AdaptiveCards/pull/4559). You can just remove these changes.

---
In reply to: [470860179](https://github.com/microsoft/AdaptiveCards/pull/3277#discussion_r470860179) [](ancestors = 470860179)